### ISSUE TITLE
Languages

### ISSUE THREAD

**Issue by Dante-Sarurobi** [Dante-Sarurobi/Hacker-777#5]
<!DOCTYPE html>
<html>
<head>
   <title>PART 1</title>
</head>
<header>
<nav>
<a href="https://youtu.be/sVHL9eOOxWY"> YOUTUBE ME </a>
<body bgcolor="white">
<h1> HELLO GUYS ITS DANTE AGAIN </h1>
<p style="font-size:30px"> So last time we discussed on joining us and stuff and now we want to discuss (once more) the best programmimg languages. Iam going to lost them in the order of easiest,hardest and best as well </p>
<h1 style="color:red"><p style="font-size=40px"><strong> EASIEST </strong></h1></p>
<p style="font-size:30px"> NUMBER 1 </p>
<p style="font-size:20px"></i> Number 1 under the genre of easiest is <a href="Download Python turorial by Codeliber"> PYTHON mobile</a>,Python is both a very powerful application as well as a comprehensive and readable programming language,the text is in <strong>ACTUAL ENGLISH</strong> which is vey hard to find in programming languages which kist speak binary code(uncomprehensive dots and numbers) which is hard to understand </i></p>
<p style="font-size=30px"><strong><marquee bgcolor="lightblue">NUMBER 2</p></marquee>
<h1 style="font-size=30px"><i> Number 2 under easiest(though it isn't a language) html,but to make it a language or , something close to a language, we have to combine tt with CSS and JAVASCRIPT,they are(if you're serious) Easy enough that you can learn with few questions a relatively smooth learning</p></i>
Just like with html 5 <a href="Codeliber apps"> CODELIBER </a> has an app or apos for both CSS and JavaScript which yiu can get here,<a href="Download CSS tutorial by CODELIBER"> CSS </a> |<a href="Download javascript tutorial by CODELIBER"> JAVASCRIPT</a>
Those are just the mobile apps you can use for learning CSS and JavaScript</p>
<h1 style="font-size:30px"><marquee bgcolor="cyan" behavior="scroll"> BEST</marquee></h1>
<p> Unfortunately those once are the fairly easy ones,now l will start to list the best and most used.
<video src="LANGS.mp4" height="300" width="300" autoplay controls="on">
  This is a screen recording on the best programmimg languages since some of you are too busy or lazy to google l did it for you </p>
</body>
</html>![Hacker.jpg](https://user-images.githubusercontent.com/81441227/112768976-f8a65100-901e-11eb-99ec-9df0c90dde1c.jpg)

**Comment by androyt1** [comment]
I am surprise.you consider phyton easier than html

**Comment by Dante-Sarurobi** [comment]
Yeah,HTML took me 4 days but Python took a week for me because of all the
math
